js/contest_edit.js

```diff
@@ -1,3 +1,10 @@
+function getDuration($obj){
```

Comment:
I see no need for it to be in global.js, since it's only used when creating/editing contests